tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/four_photo_collage_block.html

```diff
@@ -1,9 +1,16 @@
 {% load wagtailcore_tags wagtailimages_tags %}
-<div>
-    {% for item in value.images %}
-        {% srcset_image item.image format-webp loading="lazy" fill-{100x100} alt=item.image.alt_text %}
-    {% endfor %}
+<div class="grid__four-photo-collage four-photo-collage">
+    {# Note the tabindex is important here as it makes the div focussable, meaning that the scrollbar can be operated with the keyboard #}
+    <div class="four-photo-collage__scroller" tabindex="0">
+        <div class="four-photo-collage__container">
+            {% for item in value.images %}
+                <div class="four-photo-collage__image-wrapper four-photo-collage__image-wrapper--{{ forloop.counter }}">
+                    {% srcset_image item.image format-webp loading="lazy" fill-{500x500,600x600} sizes="(max-width: 1022px) 600px, 500px" class="four-photo-collage__image" alt=item.image.alt_text %}
```

Comment:
Suggestion: The alt text wasn't showing for me - I think it needs updating to `alt=item.alt_text`

Reply:
very good spot! fixed